tests/func/test_import.py

```diff
@@ -56,6 +56,28 @@ def test_import_git_file(erepo_dir, tmp_dir, dvc, scm, src_is_dvc):
     }
 
 
+def test_import_cached_file(erepo_dir, tmp_dir, dvc, scm):
+    src = "some_file"
+    dst = "some_file_imported"
+
+    erepo_dir.scm_gen({src: "hello"}, commit="add a regular file")
+    os.remove(erepo_dir.dvc.config.config_file)
```

Comment:
Hm, I don't think this affects the test at all, because dvc import clones erepo and non-commited changes won't be cloned anyways.

Reply:
So what happens in this test currently is that dvc automatically sets default remote to erepo_dir's .dvc/cache and pulls from there. https://github.com/iterative/dvc/blob/master/dvc/external_repo.py#L91 Need to somehow disable it for this test or workaround it. 🤔

Reply:
@efiop If my understanding is correct https://github.com/iterative/dvc/blob/master/tests/dir_helpers.py#L282-L283 is the code which adds remote configuration. Or am I missing something. I didn't quite get the point here. I was thinking: if removal of config from the .dvc/config from the erepo would help to replicate the scenario, while writing the test.

Reply:
@sharidasan Oh, good point! I've totally forgotten about that one. We used to use that before https://github.com/iterative/dvc/blob/master/dvc/external_repo.py#L91 logic. But still, as you can see that file is _commited_ and here you are only removing it from the workspace, not committing it, so clone of this erepo would still have that config intact.